docs/content/guides/dagster/recommended_project_structure.mdx

```diff
@@ -0,0 +1,202 @@
+
+### For resources
+
+Make resources reusable and share them across jobs or asset groups.
+
+In this example, resources (e.g., database connections, Spark sessions, API clients, and I/O managers) are grouped in the [`resources/__init__.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/resources/__init__.py) file, where they are bound to configuration sets which vary based on environment. In complex projects, we find it helpful to make resources reusable and configured with pre-defined values via <PyObject object="configured" />. So when developing a project, instead of trying to find their own way to model external components, teammates can all quickly get stuff done using a pre-defined resource set, or make changes to resources that are shared among the team.
+
+This pattern also helps you to easily execute jobs in different environments without code changes. In this example, we dynamically define a repository based on the deployment in [`repository.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/repository.py) and are able to keep all code the same across testing, local development, staging, and production. Read more about our recommendations in [Transitioning Data Pipelines from Development to Production](/guides/dagster/transitioning-data-pipelines-from-development-to-production)
+
+### For ops and graphs
+
+
+Although assets are useful in many ways, [ops](/concepts/ops-jobs-graphs/ops) and [graphs](/concepts/ops-jobs-graphs/graphs), are part of the Dagster ecosystem and can be integrated into your project structure.
+
+If you have use cases that don't meet the criteria for using assets, you may still use ops and graphs to build out these use cases. For example:
+
+- Sending an email or Slack message on a regular basis to users 
+- Reading a file of user IDs and change the value of a particular attribute for each user 
+- Scanning a data warehouse for tables that haven't been used in months and delete them
+
+In these cases, we recommend keeping all ops together in an 'ops_graphs/' directory. Similarly to assets, a structure under 'ops_graphs/' can be used to group jobs and their respective graphs into distinct business domains.  
+
+
+### For multiple repositories
+
+So far, we've discussed our recommendations for structuring a large project which contains only one repository. Dagster also allows you to structure a project with multiple Dagster repositories. We don't recommend over-abstracting too early, and in most cases, one repository should be sufficient. The pattern we found useful is to use multiple repositories is to keep conflicting dependencies separate, where each Dagster repository can keep their own package requirements (e.g., `setup.py`) and deployment specs (e.g., `Dockerfile`).
+
+### For other folders
+
+#### For tests
```

Comment:
🔧 Let's put these subsections into a tab component. As the **For other projects outside Dagster** section is (at least currently) only an example file tree, we can save space + visual load by putting it into tabs.

Ex:

```
<TabGroup>
    <TabItem name="For tests">
    Test content
    </TabItem> 
    <TabItem name="For projects outside Dagster">

**Markdown content is supported**...

But must be flush-left **and** have beginning and ending new lines to render correctly.
    </TabItem>
</TabGroup>
```

[This is a README update I'm working on](https://github.com/dagster-io/dagster/blob/eab3e939e912d719428887a5e7d8e234256b1cf6/docs/README.md#tabs) that goes into more detail about our available components.